articles/install-guide/csharp.md

```diff
@@ -20,19 +20,19 @@ To write C# programs, download and install the **.NET Core SDK** 2.1 or later fr
 ## Installation ##
```

Comment:
Comment for pre-requisites: the user doesn't need to install .net core here if they have already installed Q#, so I would remove this section.